src/swarm/neo/client/mixins/RequestCore.d

```diff
@@ -178,7 +178,54 @@ public template RequestCore ( RequestType request_type_, ubyte request_code,
     {
         public Args args;
 
-        public SerializableReferenceType!(Notifier) notifier;
+        /***********************************************************************
+
+            Note: this struct is essentially deprecated. It is only needed in
+            order to maintain the public API of UserSpecifiedParams (specifically
+            the ability to call `params.notifier.set()`). In the next major
+            release, the `serialized_notifier` can be moved into
+            UserSpecifiedParams and this wrapper struct removed.
```

Comment:
I used the keyword "deprecated" deliberately. We search for that way more often than "TODO" or "FIXME" ;)